camera-viewer/main.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const inputIPaddress = document.getElementById("ip-address")
+const toggleButton = document.getElementById("toggle-button")
```

Comment:
What do you think about `toggleButton` name?

Reply:
I often use "toggleButton", but I should have thought more carefully.